Lib/distutils/dist.py

```diff
@@ -484,6 +485,8 @@ def parse_command_line(self):
             if args is None:            # user asked for help (and got it)
                 return
 
+        log.set_verbosity(self.verbose)
```

Comment:
I think this seems a good way to handle this case; you probably don't need to change it.

Reply:
The reason why I had to move `log.set_verbosity(self.verbose)` here from line 477 is because `self.verbose` now can be set by `_parse_command_opts` if it is included as a command option instead of a global option.

In general I found it a little bit confusing the state management of this class and I think it can be improved, but for now this update was enough to let the test pass.